src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -261,6 +261,8 @@
 
   <ItemGroup>
     <Compile Include="$(CommonTestPath)System\GenericMathHelpers.cs" Link="Common\System\GenericMathHelpers.cs" />
+    <Compile Include="System\Numerics\GenericMathDimHelpers.cs" />
+    <Compile Include="System\Numerics\IBinaryNumberTests.cs" />
     <Compile Include="System\ByteTests.GenericMath.cs" />
     <Compile Include="System\CharTests.GenericMath.cs" />
     <Compile Include="System\DecimalTests.GenericMath.cs" />
```

Comment:
@tannergooding does this placement look correct now?

Reply:
LGTM.